services/QuillLMS/db/migrate/20251211142000_update_teacher_premium_plan_pricing.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class UpdateTeacherPremiumPlanPricing < ActiveRecord::Migration[7.1]
+  INTERVAL = 'yearly'
+  AUDIENCE = 'teacher'
+  DISPLAY_NAME = 'Teacher Premium'
+  NAME = 'Teacher Paid'
+  LEGACY_NAME = 'Teacher Paid Legacy'
+  INTERVAL_COUNT = 1
+  LEGACY_PRICE = 8000
+  NEW_PRICE = 11500
+
+  def up
+    Plan
+      .where(name: NAME, display_name: DISPLAY_NAME, price: LEGACY_PRICE, audience: AUDIENCE, interval: INTERVAL)
+      .update_all(name: LEGACY_NAME)
+
+    Plan.create!(
+      name: NAME,
+      display_name: DISPLAY_NAME,
+      price: NEW_PRICE,
+      audience: AUDIENCE,
+      interval: INTERVAL,
+      interval_count: INTERVAL_COUNT
```

Comment:
I suspect this doesn't matter for our purposes, but I figured I'd flag: there is a theoretical data state (one we shouldn't be in, hence it not mattering), but the `where` and `create!` don't directly overlap (`create` has `interval_count` as an argument), so you could, in theory, fail to update the legacy value and create the new one.  In practice, I don't think this will ever be an issue for us.